components/navbar/navbar.tsx

```diff
@@ -58,7 +64,7 @@ const Navbar: NextPage<Props> = ({userName, image}) => {
         onClose={onClose}
       >
         <DrawerOverlay />
-        <DrawerContent background='teal'>
+        <DrawerContent backgroundColor={bgNavbar}>
```

Comment:
This function must be called.
```suggestion
        <DrawerContent backgroundColor={bgNavbar()}>
```

This is the build error.